TileStache/Goodies/VecTiles/transform.py

```diff
@@ -3185,3 +3185,14 @@ def elevation_to_meters(shape, props, fid, zoom):
 
     props['elevation'] = _to_float_meters(elevation)
     return shape, props, fid
+
+
+def add_is_bicycle_route(shape, props, fid, zoom):
+    """
+    If the props contain a bicycle_network tag or cycleway, it should
+    have an is_bicycle_route boolean.
+    """
+    props.pop('is_bicycle_route', None)
+    if 'bicycle_network' in props or 'cycleway' in props:
```

Comment:
This only works when we exclude `no` from the property values of `cycleway` & etc before we get here? Where does that happen?

I think this should also include tests for:

* cycleway_left
* cycleway_right
* cycleway_both

Why we have 4 properties for all these instead of the 1 is a little weird, but I think just inherited from OSM?